sphinx_gallery/scrapers.py

```diff
@@ -353,7 +353,8 @@ def save_figures(block, block_vars, gallery_conf):
     return all_rst
 
 
-def figure_rst(figure_list, sources_dir, fig_titles='', srcsetpaths=None):
+def figure_rst(figure_list, sources_dir, fig_titles='', srcsetpaths=None,
+               svg_image=False):
```

Comment:
Might as well add these because it's easy
```suggestion
               *, svg_image=False):
```